src/helpers/common.js

```diff
@@ -11,15 +11,20 @@ import * as H from './';
 import locales from '../locale';
 // /////////////////////////////////////////////////////////////////////////////////////////////////
 
-export const getLocaleName = () =>
-  H.ifElse(
+export const getLocaleName = () => {
+  const localeString = H.ifElse(
     H.isNotNilAndNotEmpty(H.getItemFromLocalStorage('localeName')),
```

Comment:
create const localeNameFromLocalStorage = H.getItemFromLocalStorage('localeName') and use